### PR TITLE
Feature/card swiper

### PR DESCRIPTION
**Pull request by RomiKMoveo**
Enter description here

Enter changes made here

Enter related issues here

I have:
- [ ] deleted all irrelevant comments, logs and debuggers.
- [ ] assure that I don't print full error stacks, full body response or headers from the server in the console.
- [ ] deleted all sensitive information from the code because it's being imported from environment variables.
- [ ] encrypted keys/passwords that are sent through the internet.
- [ ] sanitized all user inputs to prevent attacks.
- [ ] made sure my code is generic and reusable for future things or reused existing resources.
- [ ] made sure that my code is readable.
- [ ] checked that the functions and resource names are self-explanatory.
- [ ] tested the code locally and it works as expected.
- [ ] added validations and error handling in relevant cases (external requests, integrations, etc).

### CODE REVIEW

File: epicure-frontend/src/pages/home/components/Chef/Chef.scss [hunk 1]
```diff
@@ -21,12 +21,60 @@
...
+            flex-direction: row;
+            gap: 60px;
+        }
+
+        img {
+            height: 262px;
+            
+            @media (min-width: $desktop-viewport-width) {
+
+                // min-width: 478px;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/pages/home/components/Chef/Chef.scss:39]: Remove comments

File: epicure-frontend/src/shared/components/SearchBar/SearchBar.scss [hunk 2]
```diff
@@ -4,7 +4,7 @@
 .search-bar {
   display: flex;
   align-items: center;
-  border-radius: 1.0vw;
+  border-radius: 0.5vw;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/shared/components/SearchBar/SearchBar.scss:7]: why is the border radius needs to be effected by the width of the screen

**Review by MoranOshiaMoveo — CHANGES_REQUESTED** [review]

File: epicure-frontend/src/shared/components/SearchBar/SearchBar.scss (hunk 2 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/shared/components/SearchBar/SearchBar.scss:7]: done

File: epicure-frontend/src/pages/home/components/Chef/Chef.scss (hunk 1 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/pages/home/components/Chef/Chef.scss:39]: done

**Pull request merged**